imageroot/systemd/user/reports-ui.service

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Reports UI Service
+Requires=reports-api.service
+After=reports-api.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=%S/state/environment
+WorkingDirectory=%S/state
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run \
+        --cidfile=%t/%n.ctr-id \
+        --cgroups=no-conmon \
+        --rm \
+        --sdnotify=conmon \
+        --detach \
+        --replace \
+        --env APP_HOST=10.0.2.2 \
+        --env APP_PORT=${REPORTS_API_PORT} \
+        --network slirp4netns:allow_host_loopback=true \
```

Comment:
Why not using `network=host`?
It seems to me you just allocated the port in the configure: https://github.com/nethesis/ns8-nethvoice/pull/52/files?diff=unified&w=1#diff-a1fba544737e3e8df1fada336a382cc171023655c91a21dc834f0a4ffed66fb5R148

Reply:
If I can I'll avoid the `host` part due to the need to customize the image below to listen to specific open ports. It's the orchestrator's job to manage port redirection when needed, this is why avoiding the `host` when possible is a good approach.

Reply:
Since it has been already tested, do not change it.